ExilenceNextApp/src/store/settingStore.ts

```diff
@@ -12,6 +12,8 @@ export class SettingStore {
   @persist
   @observable
   uiScale: number = electronService.webFrame.getZoomFactor() * 100;
+  @persist @observable logPath: string =
+    'C:/Program Files (x86)/Grinding Gear Games/Path of Exile/logs/Client.txt';
```

Comment:
As PoE supports Mac now we prooly want to check if `process === darwin` and add similar path to Mac place. Not sure where those logs are placed in that OS though.

Additionally this will work only for people with standalone client. Anyone who uses Steam/Epic Games clients won't be able to use those settings. 

Ultimately I believe we should allow people to pick a location where their Client.txt is for maximum flexibility. Though defaults like the above could be a good suggestion for any further iterations and during next release strictly mention that this feature only applies to standalone users.

Reply:
Good input agreed that this should be a config value that the user picks in the end but for testing purposes, I hardcoded the path used by the standalone client. My thought is to leave this in for now for easier testing since we still need to build a UI around the functionality before doing any release, this PR is mostly prepping the background process.